lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -117,12 +165,28 @@ next_hop_route_map:
   config_set_append: '<state> nexthop route-map <route_map>'
   default_value: ""
 
+next_hop_route_policy:
+  cli_ios_xr:
+    config_set_append:
+      - 'address-family <afi> <safi>'
+      - '<state> nexthop route-policy <route_policy>'
+    config_get: 'show running-config router bgp'
+    config_get_token: '/^router bgp <asnum>$/'
+    config_get_token_append: '/nexthop route-policy (.*)$/'
```

Comment:
seems like this should inherit from the template - or else you are missing an address-family match line?